components/skeleton/demo/button.md

```diff
@@ -0,0 +1,21 @@
+---
+order: 2
+title:
+  zh-CN: 骨架按钮
+  en-US: Skeleton Button
+---
+
+## zh-CN
+
+骨架按钮。
+
+## en-US
+
+Skeleton Button.
+
+```jsx
+import { Skeleton } from 'antd';
+
+const { Button } = Skeleton;
+ReactDOM.render(<Button>Button</Button>, mountNode);
```

Comment:
I'd prefer `<Skeleton.Button><Button {...someProps} /></Skeleton.Button>`. Since user can use HOC to customize Button. antd Button should not embed.

Reply:
But I think if we do that, it seem different with #18237.

Reply:
Discussed. `<Skeleton.Button />` directly and not wrap real Button inside. In real case use usually works like:

```jsx
return loading ? (
  <div>
    <Skeleton.Avatar />
    <Skeleton.Paragraph />
    <Skeleton.Button />
  </div>
) : (
  <div>
    <Avatar />
    <Paragraph />
    <Button />
  </div>
)
```

Independent Skeleton component not need to have `loading` prop.

Reply:
Thank you. Got it. So we need remove the loading prop and just keep the size and shape props.
And we do not need support HOC to customize element into the skeleton. Right?

Reply:
> Discussed. `<Skeleton.Button />` directly and not wrap real Button inside. In real case use usually works like:
> 
> ```js-jsx
> return loading ? (
>   <div>
>     <Skeleton.Avatar />
>     <Skeleton.Paragraph />
>     <Skeleton.Button />
>   </div>
> ) : (
>   <div>
>     <Avatar />
>     <Paragraph />
>     <Button />
>   </div>
> )
> ```
> 
> Independent Skeleton component not need to have `loading` prop.

Refactored.